README.md

```diff
@@ -54,3 +54,32 @@ cargo run -- --first-token-file /tmp/firsttoken.json
 
 At this point the token will be used and stored in the bot's cache. You won't
 need to use the `--first-token-file`.
+
+## Running with Docker
+It is recommended to run the bot using docker.
+
+### 1. Create `docker-compose.yml` file
```

Comment:
should we have this yml in a docker subdirectory or similar? wdyt?

Reply:
I don't think it should be included in the repo, as it would usually look a bit different and this is just a minimal example.